webfrontend/src/components/popup/ChangeNameForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { useState } from 'react';
+import './ChangeNameForm.scss'
```

Comment:
import scss files in `index.scss` instead of importing it in individual `.tsx` files.